apps/hermes/client/js/src/examples/HermesClient.ts

```diff
@@ -28,8 +28,29 @@ const argv = yargs(hideBin(process.argv))
   })
   .parseSync();
 
+function extractAuthorizationHeadersFromUrl(urlString: string): {
```

Comment:
i recommend naming it `extractBasicAuthorizationHeadersFromUrl`. Also, can you add a doc to this document to say what it does?